package.json

```diff
@@ -7,7 +7,7 @@
     "npm": ">=5.0"
   },
   "dependencies": {
-    "babel-polyfill": "~6.9.1",
+    "babel-polyfill": "6.9.1",
```

Comment:
TBH I'd just pin *every* package in `dependencies`. Seems much safer to update them in the CI controlled fashion than having some random PR bump it and run into unrelated failures. These are much less obvious and harder to debug.

Reply:
Sure, that works as well, and removes the blocker.